base/config.js

```diff
@@ -22,18 +22,26 @@ module.exports.schema = {
     message: 'should be a function or array of functions',
     validate: value => typeof value === 'function' || (isArray(value) && filter(value, f => typeof f === 'function').length === value.length)
   },
-  endpoint: {
-    defaultValue: () => 'https://notify.bugsnag.com',
-    message: 'should be a URL',
-    validate: () => true
-  },
-  sessionEndpoint: {
-    defaultValue: () => 'https://sessions.bugsnag.com',
-    message: 'should be a URL',
-    validate: () => true
```

Comment:
Shouldn't `endpoint` and `sessionEndpoint` remain on Configuration, for backwards compatibility?